### PR TITLE
ga - 4.67.0 - release notes

### PR DESCRIPTION
**Pull request by ScottSuarez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 ## 4.67.0 (Unreleased)
+FEATURES:
+* **New Data Source:** `google_**_iam_policy` ([#14662](https://github.com/hashicorp/terraform-provider-google/pull/14662))
+* **New Data Source:** google_vertex_ai_index ([#14640](https://github.com/hashicorp/terraform-provider-google/pull/14640))
```
Comment (shuyama1) [inline CHANGELOG.md:4]:
```suggestion
* **New Data Source:** `google_vertex_ai_index` ([#14640](https://github.com/hashicorp/terraform-provider-google/pull/14640))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+
+IMPROVEMENTS:
+* cloudrun: Added beta field `template.spec.columes.empty_dir` to `google_cloud_run_service` ([#14647](https://github.com/hashicorp/terraform-provider-google/pull/14647))
```
Comment (shuyama1) [inline CHANGELOG.md:7]: if this feature is beta specific, I think we should remove it from GA release note

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+* cloudrun: Added field `template.spec.containers.name` to `google_cloud_run_service` ([#14647](https://github.com/hashicorp/terraform-provider-google/pull/14647))
```
Comment (shuyama1) [inline CHANGELOG.md:8]:
```suggestion
* cloudrun: added field `template.spec.containers.name` to `google_cloud_run_service` ([#14647](https://github.com/hashicorp/terraform-provider-google/pull/14647))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+* compute: - promoted field `network_performance_config` to ga ([#14678](https://github.com/hashicorp/terraform-provider-google/pull/14678))
```
Comment (shuyama1) [inline CHANGELOG.md:9]:
```suggestion
* compute: - promoted field `network_performance_config` in `google_compute_instance` and `google_compute_instance_template` to ga ([#14678](https://github.com/hashicorp/terraform-provider-google/pull/14678))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
...
+* privateca: added support for encoding_format on CaPool ([#14663](https://github.com/hashicorp/terraform-provider-google/pull/14663))
 
+BUG FIXES:
+* apigee: increased `google_apigee_organization` timeout defaults to 45m from 20m ([#14643](https://github.com/hashicorp/terraform-provider-google/pull/14643))
+* cloudrun: fixed an issues where the system annotation "run.googleapis.com/operation-id" and "run.googleapis.com/ingress" may cause permadiffs for `metadata.annotation` in `google_cloud_run_service` ([#14642](https://github.com/hashicorp/terraform-provider-google/pull/14642))
+* container: fixed a crash when all zero values are specified `google_container_node_pool.upgrade_settings.blue_green_settings.standard_rollout_policy` subfields ([#14693](https://github.com/hashicorp/terraform-provider-google/pull/14693))
+* gkeonprem: updated `hostname` field of ip_block from required to optional from resource `google_gkeonprem_vmware_cluster` ([#14690](https://github.com/hashicorp/terraform-provider-google/pull/14690))
+* provider: fixed an issue where mtls transports were not used consistently ([#14621](https://github.com/hashicorp/terraform-provider-google/pull/14621))
```
Comment (shuyama1) [inline CHANGELOG.md:21]:
```suggestion
```
this is released in v4.66.0

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,24 @@
...
+* compute: - promoted field `network_performance_config` to ga ([#14678](https://github.com/hashicorp/terraform-provider-google/pull/14678))
+* compute: added `guest_os_features` and `licenses` fields to `google_compute_disk` and `google_compute_region_disk` ([#14660](https://github.com/hashicorp/terraform-provider-google/pull/14660))
+* datastream: added `mysql_source_config.max_concurrent_backfill_tasks` field to `google_datastream_stream` resource ([#14639](https://github.com/hashicorp/terraform-provider-google/pull/14639))
+* firebase: added additional import formats for app resources, removed the need to supply duplicate projects in some cases ([#14638](https://github.com/hashicorp/terraform-provider-google/pull/14638))
+* notebooks: added update-in-place support for `google_notebooks_instance.metadata` field ([#14650](https://github.com/hashicorp/terraform-provider-google/pull/14650))
+* privateca: added support for encoding_format on CaPool ([#14663](https://github.com/hashicorp/terraform-provider-google/pull/14663))
```
Comment (shuyama1) [inline CHANGELOG.md:14]:
```suggestion
* privateca: added support for `encoding_format` to `google_privateca_ca_pool` ([#14663](https://github.com/hashicorp/terraform-provider-google/pull/14663))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,4 +1,24 @@
 ## 4.67.0 (Unreleased)
+FEATURES:
+* **New Data Source:** `google_**_iam_policy` ([#14662](https://github.com/hashicorp/terraform-provider-google/pull/14662))
+* **New Data Source:** `google_vertex_ai_index` ([#14640](https://github.com/hashicorp/terraform-provider-google/pull/14640))
+
+IMPROVEMENTS:
+* cloudrun: added field `template.spec.containers.name` to `google_cloud_run_service` ([#14647](https://github.com/hashicorp/terraform-provider-google/pull/14647))
+* compute: - added field `network_performance_config` to `google_compute_instance` and `google_compute_instance_template` ([#14678](https://github.com/hashicorp/terraform-provider-google/pull/14678))
```
Comment (roaks3) [inline CHANGELOG.md:8]:
```suggestion
* compute: added field `network_performance_config` to `google_compute_instance` and `google_compute_instance_template` ([#14678](https://github.com/hashicorp/terraform-provider-google/pull/14678))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,4 +1,24 @@
...
+* privateca: added support for `encoding_format` to `google_privateca_ca_pool` ([#14663](https://github.com/hashicorp/terraform-provider-google/pull/14663))
+
+BUG FIXES:
+* apigee: increased `google_apigee_organization` timeout defaults to 45m from 20m ([#14643](https://github.com/hashicorp/terraform-provider-google/pull/14643))
+* cloudrun: fixed a permadiffs for `metadata.annotation` in `google_cloud_run_service` ([#14642](https://github.com/hashicorp/terraform-provider-google/pull/14642))
+* container: fixed a crash scenario in `google_container_node_pool` ([#14693](https://github.com/hashicorp/terraform-provider-google/pull/14693))
+* gkeonprem: updated `hostname` field of ip_block from required to optional from resource `google_gkeonprem_vmware_cluster` ([#14690](https://github.com/hashicorp/terraform-provider-google/pull/14690))
+* serviceusage: added retries to handle internal error: type: "googleapis.com" subject: "160009" when activating services ([#14727](https://github.com/hashicorp/terraform-provider-google/pull/14727))
```
Comment (roaks3) [inline CHANGELOG.md:20]:
```suggestion
* serviceusage: added retries to handle `internal error: type: "googleapis.com" subject: "160009" when activating services` ([#14727](https://github.com/hashicorp/terraform-provider-google/pull/14727))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,4 +1,24 @@
...
+* cloudresourcemanager: added retries to handle internal error: type: "googleapis.com" subject: "160009" ([#14727](https://github.com/hashicorp/terraform-provider-google/pull/14727))
```
Comment (roaks3) [inline CHANGELOG.md:21]:
```suggestion
* cloudresourcemanager: added retries to handle `internal error: type: "googleapis.com" subject: "160009"` ([#14727](https://github.com/hashicorp/terraform-provider-google/pull/14727))
```

**Review by roaks3 — APPROVED** [review]

**Pull request merged**